File: server/bedrock/src/NovaEdge/Commands/NovaCommand.php
<?php
// ═══════════════════════════════════════════════════════════════════
//  Nova Edge — Commands/NovaCommand.php
//  /novaedge (alias: ne) — join/start/status برای مچ‌های بات‌دار
// ═══════════════════════════════════════════════════════════════════
declare(strict_types=1);

namespace NovaEdge\Commands;

use NovaEdge\NovaEdgePlugin;
use pocketmine\command\Command;
use pocketmine\command\CommandSender;
use pocketmine\player\Player;

final class NovaCommand extends Command
{
    public function __construct(private NovaEdgePlugin $plugin)
    {
        parent::__construct('novaedge', 'Nova Edge: مچ‌ها و بات‌های تطبیقی', '/novaedge <join|start|status|tiers> [args]', ['ne']);
    }

    public function execute(CommandSender $sender, string $commandLabel, array $args): void
    {
        $sub = strtolower($args[0] ?? 'status');
        switch ($sub) {
            case 'tiers':
                foreach ($this->plugin->tiersSpec()['tiers'] ?? [] as $t) {
                    $sender->sendMessage(sprintf('%s — skill %s | model %s | llm %s', $t['id'], $t['skill'] ?? '?', $t['model'] ?? '—', !empty($t['llm_enabled']) ? 'on' : 'off'));
                }
                return;
            case 'status':
                $sessions = $this->plugin->sessions();
                $sender->sendMessage('§6Nova Edge§r — مچ‌های فعال: ' . count($sessions));
                foreach ($sessions as $s) {
                    $sender->sendMessage(sprintf('  %s | %s | humans %d bots %d | tier %s | %ds', $s->id(), $s->modeId(), $s->humanCount(), $s->botCount(), $s->currentTier(), $s->elapsedSec()));
                }
                $rt = $this->plugin->runtime();
                if ($rt) {
                    $sender->sendMessage('  ورکر: ' . ($rt['server_name'] ?? '?') . ' | crossplay ' . (!empty($rt['crossplay']) ? 'on' : 'off'));
                }
                return;
            case 'start':
                if (!$sender instanceof Player && !($sender->isOp() ?? false)) {
                    $sender->sendMessage('فقط از داخل بازی/با op.');
                    return;
                }
                $mode = $args[1] ?? 'bedwars';
                $fill = max(1, min(16, (int) ($args[2] ?? 4)));
                $humans = [];
                if ($sender instanceof Player) {
                    $prof = $this->plugin->profile($sender->getName()) ?? ['id' => $sender->getName(), 'rating' => 1000, 'level' => 1, 'rank_id' => 'free', 'games' => 0];
                    $humans[] = $prof;
                }
                $s = $this->plugin->startMatch($mode, $humans, $fill, random_int(1, 2 ** 31 - 1));
                $sender->sendMessage($s ? '§aمچ ' . $s->id() . ' شروع شد (tier ' . $s->currentTier() . ')' : '§cمود نامعتبر یا پل قطع است.');
                return;
            case 'join':
            default:
                $sender->sendMessage('دستور: /novaedge <status|start <mode> [bots]|tiers>');
        }
    }
}

File: server/java/src/main/java/ir/novaedge/commands/NovaCommand.java
// ═══════════════════════════════════════════════════════════════════
//  Nova Edge — commands/NovaCommand.java
//  /novaedge (و /ne): وضعیت، صف، شروع مچ، آمار بات‌ها
// ═══════════════════════════════════════════════════════════════════
package ir.novaedge.commands;

import ir.novaedge.NovaEdgePlugin;
import ir.novaedge.game.GameSession;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class NovaCommand implements CommandExecutor {

    private final NovaEdgePlugin plugin;

    public NovaCommand(NovaEdgePlugin plugin) {
        this.plugin = plugin;
    }

    @Override
    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
        String sub = args.length > 0 ? args[0].toLowerCase() : "status";
        switch (sub) {
            case "status" -> {
                sender.sendMessage("§6[NovaEdge] §fسرور: " + plugin.bridge().serverId()
                        + " | متصل: " + (plugin.bridge().configured() ? "§aبله" : "§cنه (آفلاین)")
                        + " | جلسات فعال: §b" + plugin.activeSessions().size());
            }
            case "join", "queue" -> {
                if (!(sender instanceof Player p)) {
                    sender.sendMessage("§cفقط بازیکن می‌تواند وارد صف شود.");
                    return true;
                }
                String mode = args.length > 1 ? args[1] : "bedwars";
                plugin.joinQueue(p.getUniqueId().toString(), p.getName(), mode);
                p.sendMessage("§a[NovaEdge] §fوارد صف §b" + mode + " §fشدی.");
            }
            case "start" -> {
                String mode = args.length > 1 ? args[1] : "bedwars";
                GameSession s = plugin.startLocalMatch(mode);
                sender.sendMessage(s != null
                        ? "§a[NovaEdge] §fمچ §b" + mode + " §fشروع شد (id=" + s.id() + "، بات=" + s.botCount() + ")."
                        : "§c[NovaEdge] مود ناشناخته: " + mode);
            }
            case "sessions" -> {
                if (plugin.activeSessions().isEmpty()) {
                    sender.sendMessage("§7جلسهٔ فعالی نیست.");
                }
                for (GameSession s : plugin.activeSessions().values()) {
                    sender.sendMessage("§6[NovaEdge] §f" + s.id() + " — مود: §b" + s.modeId()
                            + " | تیم‌ها: §b" + s.teams().size()
                            + " | تیر: §b" + s.currentTier()
                            + " | سپری‌شده: §b" + s.elapsedSec() + "s");
                }
            }
            case "reload" -> {
                plugin.reloadConfig();
                sender.sendMessage("§a[NovaEdge] §fکانفیگ بازخوانی شد (برای اتصال جدید، سرور را ری‌استارت کنید).");
            }
            default -> sender.sendMessage("§e/novaedge §f<status|join <mode>|start <mode>|sessions|reload>");
        }
        return true;
    }
}
